### PR TITLE
Open proposal:  filecoin-miner-map

### PR DESCRIPTION
**Pull request by koinbros-filecoin**
We just submited an open grant proposal.

### CODE REVIEW

**Comment by koinbros-filecoin** [comment]
This PR should be labeled as: category:app-dev and proposal-type:open 
@pooja Please help us to add labels since we don't have the permission, thanks!

**Comment by koinbros-filecoin** [comment]
@eshon Thank you. Any reason we did not enter wave2?

File: open-grant-proposals/filecoin-miner-map.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Our advisors include a tenure financial statistics Professor from Hong Kong University of Science and Technology (Dr. Ying Ying Li, University of Chicago, Statistics Ph.D. 2008), a hedge fund manager (Richard Warfield, HBS, EECS Berkeley) and a successful entrepreneur (Gigi Wang, Stanford).
+
+## Team code repositories
+
+<!--
+Please provide links to your team's prior code repos for similar or related projects.
+-->
+
+- [Stellar Rails](https://github.com/cryptomover-software/stellar-rails-wallet) - an open source web browser based Stellar wallet
+
```
Comment (eshon) [inline open-grant-proposals/filecoin-miner-map.md]: Hi @koinbros-filecoin,

Do you have any examples of projects you've built that are live that we can see?

Comment (gavinito) [reply open-grant-proposals/filecoin-miner-map.md]: yep, the wallet we have built is still being hosted on www.cryptomover.tech

Comment (eshon) [reply open-grant-proposals/filecoin-miner-map.md]: Hi @gavinito, 
Your proposal budget is really large and your team doesn’t have any evidence of d3 or libp2p knowledge for the network QoS related items. Not sure a Stellar Rails wallet is enough previous work. This is not likely to get funded. Perhaps you could propose a project with a much smaller budget to start, such as in the $5k range as a proof of concept.

**Comment by eshon** [comment]
Greetings,

Thank you for your application to the Filecoin DevGrants program. We appreciate your proposal!

Your team was not selected for this grant in Wave 2. 

Please stay tuned for more Filecoin dev grant RFPs in Wave 3 whose deadline is March 31st.

If you're interested in applying again in the future, note that we will fund teams with a proven track record of delivering high-quality software projects (preferably open source).

In the case of a team we're not familiar with or with less of an open source track record, we prefer smaller proof-of-concept projects that demonstrate your team's ability to learn about Filecoin and build something unique to the ecosystem.

**Pull request closed without merging**